package.json

```diff
@@ -26,11 +26,9 @@
     "coverage:test:default": "shx cat ./coverage/default.text.txt && shx cat ./coverage/default.text.summary.txt",
     "test:legacy": "karma start karma.legacy.conf.js",
     "coverage:test:legacy": "shx cat ./coverage/legacy.text.txt && shx cat ./coverage/legacy.text.summary.txt",
-    "server:test:start": "go run test/server/server.go &",
-    "server:test:stop": "go run test/server/shutdown.go",
-    "test": "run-s lint server:test:start build test:* server:test:stop coverage:test:*",
+    "test": "run-s lint build test:* coverage:test:*",
     "clean": "shx rm -rf *.log lib/ coverage/",
-    "lint": "eslint src/**/*.js test/**/*.js",
+    "lint": "eslint src/**/*.js",
```

Comment:
@leodido because it doesn't what you can think it does.
It can seem that tests are linted correctly, but it's true only for the first level of directories.
In a future PR I'll lint all the files under "test" folder.